src/backends/torch/torchlib.cc

```diff
@@ -125,9 +127,98 @@ namespace dd
   {
   }
 
+  void TorchModule::to(torch::Device device)
+  {
+    if (_graph)
+      _graph->to(device);
+    if (_native)
+      _native->to(device);
+    if (_traced)
+      _traced->to(device);
+  }
+
+  void TorchModule::to(torch::Dtype dtype)
+  {
+    if (_graph)
+      _graph->to(dtype);
+    if (_native)
+      _native->to(dtype);
+    if (_traced)
+      _traced->to(dtype);
+  }
+
+  void TorchModule::to(torch::Device device, torch::Dtype dtype)
+  {
+    if (_graph)
+      _graph->to(device, dtype);
+    if (_native)
+      _native->to(device, dtype);
+    if (_traced)
+      _traced->to(device, dtype);
+  }
+
+  template <class TInputConnectorStrategy>
+  void TorchModule::post_transform(const std::string tmpl,
```

Comment:
we may call this function `load` for clarity ?

Reply:
in my mind it was more like a hook (that loads in case of lstm and nbeats) or does nothing (in case of traced  becasue already loaded) . We should in the future clarify all this when unifying TorchModule as a graph

Reply:
OK, great, to be clarified in terms of API in later updates!